.github/workflows/jacoco_report.yml

```diff
@@ -28,48 +28,71 @@ env:
   changed: 80.0
 
 jobs:
-  test:
-    name: Build and test
+  build-test-and-measure:
+    name: Build, Test and Measure
     runs-on: ubuntu-latest
 
+    services:
+      postgres:
+        image: postgres:15
+        env:
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: atum_db
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
       - name: Setup Scala
         uses: olafurpg/setup-scala@v14
         with:
-          java-version: "adopt@1.8"
+          java-version: "adopt@1.11.0-11"
+      - name: Prepare testing database
+        run: sbt flywayMigrate
       - name: Build and run tests
         continue-on-error: true
         id: jacocorun
-        run: |
-          sbt "project agent; jacoco"
-          sbt "project model; jacoco"
-      #  agent module code coverage
-      - name: Add coverage to PR
+        run: sbt jacoco
+      - name: Add coverage to PR (model)
+        if: steps.jacocorun.outcome == 'success'
+        id: jacoco-model
+        uses: madrapps/jacoco-report@v1.6.1
+        with:
+          name: model-jacoco-report
+          paths: ${{ github.workspace }}/model/target/jvm-${{ env.scalaShort }}/jacoco/report/jacoco.xml
+          token: ${{ secrets.GITHUB_TOKEN }}
+          min-coverage-overall: ${{ env.overall }}
+          min-coverage-changed-file: ${{ env.changed }}
```

Comment:
Just like Ladislav said, I think this `min-coverage-changed-file` should be changed to `min-coverage-changed-files`